apps/react-app/src/common/forms/mf-button-component.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { cx } from '@emotion/css';
+import * as classes from './mf-button-styles';
+
+type Props = React.ButtonHTMLAttributes<HTMLButtonElement>;
+
+export const ButtonComponent: React.FC<Props> = props => {
```

Comment:
Naming is always hard, and there can be several valid options.

I think could use the following

MfButton

Reply:
Done.